docs/SETUP.md

```diff
@@ -0,0 +1,227 @@
+# Quickstart
+
+Make a copy of `.env_sample` called `.env` in the root of the project directory. Confirm settings,storage,SSL and ports.
+
+Run the dev_setup script to install all dependencies and create a new virtual environment (venv) for CodaLab:
+
+1. **Windows Users** do 
+    `cd codalab`
+    `.\dev_setup.bat`
+
+2. **Linux Users** do 
+    `cd codalab`
+    `source ./dev_setup.sh`
+
+Activate your environment:
+
+1. **Windows Users** do 
+    `venv\Scripts\activate`
+
+2. **Linux Users** do 
+    `source venv/bin/activate`
+
+Navigate to the root of the project directory, or where `docker-compose.yml` lives, and do:
+
+`docker-compose up -d`
+
+to bring up the project and all of its services. You may now do:
+
+`docker logs -f <service_name>`
+
+to view the state of your services, where `<service>` is the name of the service.
+
+
+# Setup
+
+## On *nix
```

Comment:
I've not seen this before, I'm thinking that it just means all unix based operating systems, but it's a little confusing to the uninitiated. How about just specifying a little more basic? Or specify that this is for Mac and Linux and all Unix based OS?